cmd/power/src/lib.rs

```diff
@@ -137,38 +231,69 @@ fn power(context: &mut ExecutionContext) -> Result<()> {
         }
     }
 
+    //
+    // If we have been asked for phase currents, we'll get all of those now.
+    //
+    if subargs.phase_current {
+        phase_currents(hubris, core, &mut context, &mut devices)?;
+    }
+
     println!(
-        "{:25} {:>8} {:>8} {:>8} {:>8} {:>8}",
+        "{:30} {:>8} {:>8} {:>8} {:>8} {:>8}",
         "RAIL", "VOUT", "IOUT", "VIN", "IIN", "TEMP"
     );
 
+    let no = "-";
+    let err = "x";
+
     let p = |what| {
         print!(" ");
 
         match what {
             Some(ndx) => {
                 if let Some(value) = rval[ndx] {
-                    print!("{:>8.2}", value);
+                    print!("{value:>8.3}");
                 } else {
-                    print!("{:>8}", "-");
+                    print!("{err:>8}");
                 }
             }
             None => {
-                print!("{:8}", "");
+                print!("{no:>8}");
             }
         }
     };
 
     for d in devices.values() {
-        print!("{:25}", d.name);
+        print!("{:30}", d.name);
 
         p(d.voltage);
         p(d.current);
+
         p(d.input_voltage);
         p(d.input_current);
         p(d.temperature);
 
         println!();
+
+        if let Some(phase_currents) = &d.phase_currents {
+            if phase_currents.len() > 1 {
```

Comment:
This means we don't print phase current for the ISL68224, which only has one phase per rail.  Is that intentional?

Reply:
Yes, that's intentional.